guides/common/attributes-titles.adoc

```diff
@@ -3,7 +3,7 @@
 :AdministeringDocTitle: Administering {ProjectName}
 :APIDocTitle: Using the {Project} REST API
 :AppCentricDeploymentDocTitle: Deploying hosts by using application centric approach
-:ConfiguringDNSDHCPTFTPDocTitle: Configuring DNS, DHCP, and TFTP integration
+:IntegratingProvisioningInfrastructureServices: Integrating provisioning infrastructure services
```

Comment:
```suggestion
:IntegratingProvisioningInfrastructureServicesDocTitle: Integrating provisioning infrastructure services
```

What about DocURL? Please keep them in sync & alignment to the current naming scheme.

Reply:
I updated the DocURL attribute as well.